forkrun.bash

```diff
@@ -1163,44 +1163,55 @@ p_PID+=(\${p{<#>}_PID})""" )"
             while ${continueFlag}; do
```

Comment:
This is a good idea, but has 1 minor issue - it always waits for there to be 128 output files before printing any of them. This can cause a lot of unneeded latency waiting for output if, say, you have 127 output files ready and the 128th is taking a long time to run. Ideally, whenever output can be printed (in the correcvt order) all of it is printed, and this printing is grouyped into groups of no mores thanb 128 files.

I added a new commit that implements this and combines your suggestion with the existing code.